AnkiDroid/src/main/java/com/ichi2/libanki/CollectionV16.kt

```diff
@@ -156,6 +156,27 @@ class CollectionV16(
         return cardIdsList
     }
 
+    override fun findNotes(
+        query: String?,
+        order: SortOrder,
+    ): List<Long> {
+        val adjustedOrder = if (order is SortOrder.UseCollectionOrdering) {
+            @Suppress("DEPRECATION")
+            SortOrder.BuiltinSortKind(
+                get_config("sortType", null as String?) ?: "noteFld",
```

Comment:
```suggestion
                get_config("noteSortType", null as String?) ?: "noteFld",
```

Notes mode uses different ordering key - found in collection.py in desktop code.

Reply:
Also, does AnkiDroid expose a way to change the value of that config entry?

Reply:
For cards yes. For note no. But I assume the same code can be used depending on card or note mode.